src/Theming/ThemeSwitcher.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import * as React from "react";
+import defaultTheme from "orbit-design-token";
+
+import ThemeProvider from "./ThemeProvider";
+
+type ThemeProps = {
+  children: any,
+  mainTheme?: typeof defaultTheme,
+  themes: any[],
+};
+
+type ThemeState = {
+  themeIndex: number,
+};
+
+export default class ThemeSwitherSample extends React.Component<ThemeProps, ThemeState> {
```

Comment:
I would suggest to have the same name for default export as for the filename

Reply:
OK